models/CoronSegmentator/configs/inference.json

```diff
@@ -0,0 +1,17 @@
+{
+    "bundle_root": "./",
+    "ckpt_dir": "$@bundle_root + 'models'",
+    "output_dir": "$@bundle_root + 'Output'",
+    "output_ext": ".usd",
+    "input_file": "Input/E_CT_0000.nii.gz",
+    "infer": {
+        "_target_": "scripts.heart_digital_twin.CoroSegmentator_Pipeline",
```

Comment:
As @yiheng-wang-nv mentioned above, please include the unit test to ensure the bundle works as expected.
I didn't see this file under the scripts folder.